### PR TITLE
Enable DumpLog and Name2EE sos commands

### PR DESCRIPTION
**Pull request by mikem8361**
Also fixed expression parsing used by some commands to default to hex (without the 0x) since most of the addresses and hex values printed by the sos commands don't have the 0x.

### CODE REVIEW

**Comment by mikem8361** [comment]
@jkotas could you review?

**Comment by mikem8361** [comment]
@dotnet-bot test this please

File: src/ToolBox/SOS/Strike/stressLogDump.cpp [hunk 1]
```diff
@@ -330,13 +334,18 @@ HRESULT StressLog::Dump(ULONG64 outProcLog, const char* fileName, struct IDebugD
 
     if (bDoGcHist)
     {
+#ifdef FEATURE_PAL
+        ExtOut ("GC history not support\n");
```
Comment (jkotas) [inline src/ToolBox/SOS/Strike/stressLogDump.cpp]: not supported?

**Comment by jkotas** [comment]
LGTM (modulo minor comment). Feel free to merge after squash and green CI.

**Comment by mikem8361** [comment]
@dotnet-bot test this please

**Comment by sergiy-k** [comment]
LGTM. Thank you!

**Pull request merged**